godtools/App/Features/Tools/SwiftUI/ToolCardView/ToolCardView.swift

```diff
@@ -0,0 +1,101 @@
+//
+//  AllToolsContentView.swift
+//  godtools
+//
+//  Created by Rachael Skeath on 3/30/22.
+//  Copyright © 2022 Cru. All rights reserved.
+//
+
+import SwiftUI
+
+struct ToolCardView: View {
```

Comment:
Hey @rachaelblue I didn't catch this before, but I would like you to update the text in ToolCardView to match ToolCell.xib.  
Created a separate ticket here as I had some additional changes I'd like you to make https://jira.cru.org/browse/GT-1542.